### PR TITLE
Adds Link to OONI Blog & OONI Reports from About Page

### PR DESCRIPTION
**Pull request by aanorbel**
see ooni/probe#1850

### CODE REVIEW

**Comment by aanorbel** [comment]
# Preview on device 

| Light Mode  | Dark Mode|
| ------------- | ------------- |
| photo_2022-01-11 13 42 53 | photo_2022-01-11 13 42 58|

**Review by agrabeli — COMMENTED** [review]
Great work, thank you @aanorbel! 

I'd suggest @hellais reviews as well, as the CI checks appear to be failing.

**Comment by aanorbel** [comment]
CI failure seems to be a recurring issue. same as in ooni/probe-android#471. An issue was created(ooni/probe#1924) by @bassosimone  to investigate and resolve the problem.

**Review by bassosimone — APPROVED** [review]
🐳 thank you!

**Comment by hellais** [comment]
Thanks for putting this together. I am just seeing it now.

I think we make the links to the blog and reports, secondary action buttons similar to how the "data policy" link/button is.

That's because it's considered an anti-pattern to have more than one primary action on a certain screen.

**Comment by aanorbel** [comment]
> Thanks for putting this together. I am just seeing it now.
> 
> I think we make the links to the blog and reports, secondary action buttons similar to how the "data policy" link/button is.
> 
> That's because it's considered an anti-pattern to have more than one primary action on a certain screen.

This is quite accurate i think the option just s. I will make the necessary  update.

File: app/src/main/java/org/openobservatory/ooniprobe/activity/InfoActivity.java [hunk 1]
```diff
@@ -28,6 +28,14 @@
...
+	@OnClick(R.id.blog) void onBlogClick() {
+		startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://ooni.org/blog/")));
+	}
+
+	@OnClick(R.id.reports) void onReportsClick() {
+		startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://ooni.org/reports/")));
+	}
+
 	@OnClick(R.id.learnMore) void onLearnMoreClick() {
 		startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://ooni.io/")));
```
Comment (hellais) [inline app/src/main/java/org/openobservatory/ooniprobe/activity/InfoActivity.java:40]: While you are at it, can you also change the link here to be `ooni.org`?

Comment (aanorbel) [reply app/src/main/java/org/openobservatory/ooniprobe/activity/InfoActivity.java:40]: Just didn't know how to handle that. Will the update.

**Comment by aanorbel** [comment]
> Thanks for putting this together. I am just seeing it now.
> 
> I think we make the links to the blog and reports, secondary action buttons similar to how the "data policy" link/button is.
> 
> That's because it's considered an anti-pattern to have more than one primary action on a certain screen.


Screenshot_20220114_195915

**Review by hellais — CHANGES_REQUESTED** [review]
Can we have the links below the "learn more" button be evenly spaced? 

Currently there is much more spacing between "Blog" and "Reports" compared to "Reports" and "OONI Data Policy". I suggest using the value of spacing that currently exists between "Reports" and "OONI Data Policy" for the other elements as well.

**Comment by aanorbel** [comment]
<details>
<summary>ScreenShot of page</summary>

</p>
</details>

**Review by hellais — APPROVED** [review]
LGTM 🐳 

The CI, though, is failing, so we should document that we need to look into fixing that soon too.

**Pull request merged**